arch/arm/boot/dts/overlays/README

```diff
@@ -2444,6 +2482,13 @@ Info:   This overlay is now deprecated. Use the sd_* dtparams in the
 Load:   <Deprecated>
 
 
+Name:   seeed-can-fd-hat
+Info:   Overlay for Seeed Studio CAN BUS FD HAT with two CAN FD channels and an RTC. 
```

Comment:
This line also has an extra space at the end, and is too long (there's a strict 80-character limit).